dags/utils/dataproc.py

```diff
@@ -21,7 +21,7 @@ def __init__(self,
                  cluster_name=None,
                  num_workers=2,
                  image_version='1.4',
-                 zone='us-central1-b',
+                 zone='us-west1-b',
```

Comment:
Why are zones changed to `west` here?

Reply:
We have created the gcs buckets in that region, so we want to avoid cross region transfers. Originally I had set up us-central1-b because I had thought we had resource limits in west1-b, but I was confusing that for gke clusters.